### DISCUSSION TITLE
No limitation from selected photos

### DISCUSSION THREAD

**Discussion by Guilherme-Heibel** [dimonovdd/Xamarin.MediaGallery#40] [Q&A] [answered]
In Android...
Informing the number of photos that can be uploaded in a project that I use in your package, but it is not limiting... always informs that the maximum is 500 even if setting 50 for example... My great expectation in this package would be to be able to limit the number of selected photos... is it possible?

i using

 var  cts = new CancellationTokenSource(
                        TimeSpan.FromMinutes(5));
 MediaFileType[] types = new MediaFileType[2];

            types[0] = MediaFileType.Image;
            types[1] = MediaFileType.Video;
 pickResult = await MediaGallery.PickAsync(
                    new MediaPickRequest(50, types)
                    {
                       
                    },
                    cts.Token);

**Comment by dimonovdd** [comment, marked as answer]
Hi. This plugin uses a native picker on android. it only supports the ability to select one or more files. We can't set limits.
I tried to describe it in Readme.md file

**Reply by Guilherme-Heibel** [reply]
Ok! Tanks! It would be very exceptional if one day I managed to overcome this kind of limitation, I'm sure there are many projects interested in this option, congratulations for the project, it really is better than the MediaPlugIn...

**Reply by CodingOctocat** [reply]
Now, is the pick number limit available?

**Reply by beeradmoore** [reply]
@CodingOctocat , has anything changed in the Android SDK to allow that?

**Reply by CodingOctocat** [reply]
@beeradmoore I'm a newbie and I have no experience in mobile app development, so unfortunately I can't answer this question.

**Reply by beeradmoore** [reply]
All good :). I thought you meant something had changed and were waiting on this library to catch up.

I had a quick look at the latest Android SDKs and Androids current recommendations on how to pick images. Setting a limit is still not possible.
